cloud-gcp/src/main/java/com/sequenceiq/cloudbreak/cloud/gcp/GcpPlatformResources.java

```diff
@@ -99,6 +98,12 @@ public class GcpPlatformResources implements PlatformResources {
 
     private static final int DEFAULT_PAGE_SIZE = 50;
 
+    private static final int GCP_LOCAL_SSD_ALLOWED_VALUES = 375;
+
+    private static final Set<Integer> GCP_LOCAL_SSD_POSSIBLE_NUMBER_VALUES = Set.of(1, 2, 3, 4, 5, 6, 7, 8, 16, 24);
+
+    private static final Set<String> MACHINE_TYPES_WITH_LOCAL_SSD = Set.of("n1", "n2", "n2d");
```

Comment:
Have you considered moving these to configuration values?

Reply:
I did not.